core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/common/AbstractUserStoreManager.java

```diff
@@ -491,6 +490,14 @@ protected abstract void doUpdateRoleName(String roleName, String newRoleName)
     protected abstract String[] doListUsers(String filter, int maxItemLimit)
             throws UserStoreException;
 
+    /**
+     * Get the count of users with claims that matching the filter
+     *
+     * @return the number of users matching the filter with the claim.
+     * @throws UserStoreException UserStoreException
+     */
+    public abstract Long doCountClaims(String claimURI, String valueFilter) throws UserStoreException;
```

Comment:
If we introduce new abstract methods to AbstractUserStoreManager it will break the existing implementations. Therefore let's implement the method here to preserve the backward compatibility.

Reply:
fixed